src/main/java/redstone/multimeter/common/network/PacketWrapper.java

```diff
@@ -9,6 +10,11 @@ public record PacketWrapper(RSMMPacket packet) implements CustomPacketPayload {
 	public static final StreamCodec<FriendlyByteBuf, PacketWrapper> STREAM_CODEC = CustomPacketPayload.codec(Packets::encode, Packets::decode);
 	public static final Type<PacketWrapper> TYPE = CustomPacketPayload.createType("rsmm");
 
+	public static void register() {
```

Comment:
Why add the FAPI dependency? I don't think much (if anything) changed in networking since 1.20.5, so the old code should still work just fine.

Reply:
it is weird. i could not figure out why, but the `method_58270` mixin (src/main/java/redstone/multimeter/mixin/common/ClientboundCustomPayloadPacketMixin.java) was causing a crash when entering the world. since i ran out of ideas i just included FAPI to at least have a working 1.21 fork

Reply:
Might be that the lambda method that Mixin is targeting was renamed, or perhaps there was a tweak to that code after all. I'll check it out some time later this week.

Reply:
undoing these changes, it seems to just work - I'll just push that and the other changes I want and we'll be good to go

Reply:
oh, okay! would you mind telling me where you got the intermediary method name from? i could not find it at all when trying to build (_even in the decompiled class files from the jars_, which implies my build environment was somehow messed up but that's unrelated) and i'm wondering if there are useful modding resources i've overlooked

Reply:
I opened the class in a bytecode viewer and found it that way. Lambda methods are synthetic so you won't find them in a source file.